app/controllers/v1/sessions_controller.rb

```diff
@@ -250,16 +250,19 @@ def login_params(type)
 
     def saml_request_stats
       tracker = url_service.tracker
+      authn_context = tracker&.payload_attr(:authn_context)
       values = {
         'id' => tracker&.uuid,
-        'authn' => tracker&.payload_attr(:authn_context),
+        'authn' => authn_context,
         'type' => tracker&.payload_attr(:type),
         'transaction_id' => tracker&.payload_attr(:transaction_id)
       }
       Rails.logger.info("SSOe: SAML Request => #{values}")
+      normalized_authn = authn_context&.class == Array ? authn_context.join('_') : authn_context
```

Comment:
I checked datadog and if the `authn_context` is just a string we don't want to prepend it with `_`. You could add it up here instead. Could you add a spec for this case?

Also, you can use `is_a?` to check type:
```ruby
authn_context.is_a?(Array) 
```

Reply:
could you add a spec for the case when `authn_context` is a string? 🙏